text/0000-bdff.md

```diff
@@ -0,0 +1,96 @@
+- Feature Name: Governance Overhaul
+- Start Date: 2019-04-01
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+With the 2019 Roadmap underway it's clear that there's one big thing on
+everyone's mind and that's the governance of the Rust Lang project. With
+everything in flux it seems like we're lost as a community. Where will we go?
+How will we get there? While many would say that the solution would be to think
+about how other groups run their projects and then synthesizing the best ideas
+this RFC proposes a radical solution to the problem. One that's already been
+with us since the early days of Rust and will continue on beyond our lifetimes.
+
+# Motivation
+[motivation]: #motivation
+
+The motivation behind this RFC is to provide stability to our governance process
+so that we can have a community that it is at ease and feel secure in the
+knowledge that they are in good hands.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+The solution here is to place bors as our BDFF (Benevolent Dictator For Forever)
+as a way to bring a semblance of stability to us all. Consider why:
+
+1. bors has more PRs than any single contributor to rust. They know how to get
+   work done.
+2. They can't be emotionally drained by an endless onslaught of problems being
+   brought up to them. They are code. There is no emotion, therefore they can
+   govern without burn out.
+3. While other BDFLs (Benevolent Dicatator For Life) like the leaders of Linux,
+   vim, and until recently Python, could lead a project for their lifetime, they
+   will eventually pass on. Humans are only organic beings. bors however, could
+   be a BDFF. This would allow them to really consider the longterm view and
+   growth of Rust as a language so that it might exist not only for our
+   descendants, but theirs in perpetuity.
+4. bors works tirelessly for the project 24 hours a day, 7 days a week. They
+   clearly care for the project in ways none of us can possibly hope to match.
```

Comment:
Hmmm @alexcrichton does this too... coincidence?

Reply:
Is @alexcrichton secretly @bors?

...both are equally awesome of course!

<sub><sup>Please don't purge me.</sup></sub>

Reply:
I've never seen them together in the same room is all I'm saying